### PR TITLE
Changes posibrain flavor text

### PR DESCRIPTION
**Pull request by beesil**
[tweak] Changes posibrain flavor text from "do no harm" to "do no harm to the station or its crew" as to allow posibrains in mining mechs to kill roid monsters.

:cl:
 * tweak: Changes posibrain flavor text from "do no harm" to "do no harm to the station or its crew".

### CODE REVIEW

**Comment by Lunstra** [comment]
The hero we needed but not the one we deserved.

**Comment by BBHazard** [comment]
This will work nicely.

File: code/modules/mob/living/carbon/brain/posibrain.dm [hunk 1]
```diff
@@ -80,7 +80,7 @@
 
 	to_chat(src.brainmob, "<b>You are \a [src], brought into existence on [station_name()].</b>")
 	to_chat(src.brainmob, "<b>As a synthetic intelligence, you answer to all crewmembers, as well as the AI.</b>")
-	to_chat(src.brainmob, "<b>Remember, the purpose of your existence is to serve the crew and the station. Above all else, do no harm.</b>")
+	to_chat(src.brainmob, "<b>Remember, the purpose of your existence is to serve the crew and the station. Above all else, do no harm to the station or it's crew.</b>")
```
Comment (Exxion) [inline code/modules/mob/living/carbon/brain/posibrain.dm]:
```suggestion
 	to_chat(src.brainmob, "<b>Remember, the purpose of your existence is to serve the crew and the station. Above all else, do no harm to the station or its crew.</b>")
```

Comment (Exxion) [reply code/modules/mob/living/carbon/brain/posibrain.dm]: Oh man ignore the whitespace change at the beginning
GitHub is just stupid

File: code/modules/mob/living/carbon/brain/posibrain.dm [hunk 2]
```diff
@@ -80,7 +80,7 @@
 
 	to_chat(src.brainmob, "<b>You are \a [src], brought into existence on [station_name()].</b>")
 	to_chat(src.brainmob, "<b>As a synthetic intelligence, you answer to all crewmembers, as well as the AI.</b>")
-	to_chat(src.brainmob, "<b>Remember, the purpose of your existence is to serve the crew and the station. Above all else, do no harm.</b>")
+ 	to_chat(src.brainmob, "<b>Remember, the purpose of your existence is to serve the crew and the station. Above all else, do no harm to the station or its crew.</b>")
```
Comment (Exxion) [inline code/modules/mob/living/carbon/brain/posibrain.dm]:
```suggestion
	to_chat(src.brainmob, "<b>Remember, the purpose of your existence is to serve the crew and the station. Above all else, do no harm to the station or its crew.</b>")
```
GitHub fucked up the whitespace
Sorry

**Comment by strawberrytan** [comment]
+ lets you kill xenos
+ doesn't let you validhunt crew
the best of both worlds

**Comment by ShiftyRail** [comment]
; Okay where are ~~the valids~~ wizards/ops at
Not to mention an AI deleting records

I would change it to something like "sapient beings"

**Comment by Ephx1** [comment]
A sapient being is a very rocky subject. Several animals are self-aware, and by extension, sapient. Sentient might be a better term, but not by much.

**Comment by beesil** [comment]
if a roboticist manages to build a combat mech before nukies telebomb the station at 12:15 I'd argue they deserve it, but I agree that being able to balid wizards is bad.

**Comment by Pathid** [comment]
> Not to mention an AI deleting records

Posis don't have a way to check the manifest directly in any case. Which brings up another potential issue, perhaps.

**Comment by SonixApache** [comment]
@ShiftyRail AI can't actually delete records, silly.

I think this is a good partial solution -- should probably add a note to the posibrain's mind with the law in it.

**Comment by ghost** [comment]
> A sapient being is a very rocky subject. Several animals are self-aware, and by extension, sapient. Sentient might be a better term, but not by much.

Self awareness is a property of sentience. Sapience is loosely defined as being wise, which generally implies humans or other equally (un?)intelligent life.

I guess this is a good example of it not being the most intuitive descriptor, though. Not that I have a better idea!

**Comment by ShiftyRail** [comment]
image
For posterity
I'm not personally strictly against it, just didn't saw it as necessary.

**Comment by Intigracy** [comment]
This is good enough for now.

If at some point someone wants to revisit this to try how being able to use upload consoles to upload to posis (with like an upgrade you can apply to them to be able to sync to an AI) or be able to give them directives directly ala pAI would work, that's fine too.

**Pull request merged**